components/vf-show-more/vf-show-more.njk

```diff
@@ -34,3 +34,22 @@
   <button class="vf-button vf-button--primary vf-button--sm | vf-show-more__button" data-vf-js-show-more-button>Show more</button>
   <button class="vf-button vf-button--primary vf-button--sm | vf-show-more__button--less" data-vf-js-show-more-button--less>Show less</button>
 </section>
+
+
+{# A second set to show you can have more than one set on a page #}
+
+<section data-vf-js-show-more data-vf-js-show-more-pager-size="2" class="vf-show-more">
+  {% render '@vf-section-header--default', {
+    "section_title": "Show more demo part 2"
+  } %}
+
+  <ul class="vf-list vf-list--ordered">
+    <span class="vf-show-more__item"><li class="vf-list__item">a list item</li></span>
```

Comment:
the HTML is invalid here - the only direct child of an `ul` (or `ol`) that's valid is an `li`.

Reply:
Good catch. my css method could have been better. resolved.